Anlab.Mvc/Controllers/LabController.cs

```diff
@@ -325,6 +325,55 @@ public IActionResult ListUsersOrders(string id)
         }
 
         [Authorize(Roles = RoleCodes.Admin)]
+        [HttpGet]
+        public async Task<ActionResult> OverrideOrder(int id)
+        {
+            var order = await _dbContext.Orders.Include(i => i.Creator).SingleOrDefaultAsync(o => o.Id == id);
+
+            if (order == null)
+            {
+                return NotFound();
+            }
+
+            var model = new OrderReviewModel();
+            model.Order = order;
+            model.OrderDetails = order.GetOrderDetails();
+            model.HideLabDetails = false;
+
+            return View(model);
+        }
+
+        [Authorize(Roles = RoleCodes.Admin)]
+        [HttpPost]
+        public async Task<ActionResult> OverrideOrder(int id, Order order)
+        {
+            var orderToUpdate = await _dbContext.Orders.Include(i => i.Creator).SingleOrDefaultAsync(o => o.Id == id);
+
+            if (orderToUpdate == null)
+            {
+                return NotFound();
+            }
+
+            orderToUpdate.Paid = order.Paid;
+            orderToUpdate.Status = order.Status;
+            orderToUpdate.IsDeleted = order.IsDeleted;
+            if (order.IsDeleted)
+            {
+                ErrorMessage = "Order deleted!!!";
+            }
+
+            await _dbContext.SaveChangesAsync();
+
+            Message = "Order Updated";
+            if (order.IsDeleted)

```

Comment:
@jSylvestre why do you have the same if statement twice.  is one of them supposed to return?  And why a message plus an ErrorMessage for the isDeleted step

Reply:
My bad...